recipes/csvtk/meta.yaml

```diff
@@ -10,16 +10,16 @@ about:
   license: MIT
 
 build:
-  number: 1
+  number: 0
   skip: False
 
 source:
   url: https://github.com/shenwei356/csvtk/releases/download/v{{ version }}/csvtk_darwin_amd64.tar.gz # [osx]
-  md5: 771d0f5825ed9ee34ebdbb377ada10ed # [osx]
+  md5: a3cdc89301b92575297cf5ec84060453 # [osx]
   url: https://github.com/shenwei356/csvtk/releases/download/v{{ version }}/csvtk_linux_386.tar.gz # [linux]
-  md5: eeb18ddc864d1e1ec491cb755aea88fe # [linux]
+  md5: 519d31f4b76e5fd1d3ec500678faf727 # [linux]
   url: https://github.com/shenwei356/csvtk/releases/download/v{{ version }}/csvtk_linux_amd64.tar.gz # [linux64]
-  md5: 5d9c00daf0670f3ed6a958c77cf58fb8 # [linux64]
+  md5: 7c5a3dee098d9656ac7a0474e5422916 # [linux64]
```

Comment:
Please merge the two linux options. We only ship 64-bit, which uses the `linux` selector.

Reply:
Merged.

Reply:
Hi @epruesse , they were merged.